src/main/resources/db/changelog/20230710000000-create-anonymous-consumers.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+<!-- TODO: need to add the provider short name here and in the model/curator -->
+
+    <changeSet id="20230710000000-1" author="jalbrech">
+        <createTable tableName="cp_anonymous_cloud_consumers">
+            <column name="id" type="varchar(32)">
+                <constraints primaryKey="true" />
+            </column>
+            <column name="uuid" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cloud_account_id" type="varchar(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="cloud_provider_short_name" type="varchar(255)">
```

Comment:
The short name is always 3 or 4 characters long (AWS/ACN/AGOV/MSAZ/GCE). Let's make this column length of 10 or 15 to be sure, but 255 would be a waste

Reply:
Yeah, that is a good idea! Updated